pkg/oomstore/types/group.go

```diff
@@ -38,6 +38,10 @@ func (fg *Group) Copy() *Group {
 
 type GroupList []*Group
 
+func (l *GroupList) Len() int {
```

Comment:
`*` is not needed

Reply:
for consistency with FeatureList.Len method

Reply:
How about I change both of them?